driver-core/src/test/java/com/datastax/driver/core/policies/DowngradingConsistencyRetryPolicyIntegrationTest.java

```diff
@@ -40,10 +40,9 @@
         numberOfNodes = 3,
         config = {
                 // tests fail often with write or read timeouts
-                "hinted_handoff_enabled:true",
                 "phi_convict_threshold:5",
-                "read_request_timeout_in_ms:100000",
```

Comment:
Yes, these timeouts are insane, and yet this test and ConsistencyTest both fail very frequently with write or read timeouts where the query is expected to succeed. Raising the timeouts has definitely made them less error-prone. My best guess is that in CI we are maybe sharing CPU resources among several vms and consequently processes get stuck very frequently waiting for an available core, so I suppose the replica that is meant to process the query gets paused, and the coordinator eventually triggers a timeout.